common/setups/rasr/onnx_precomputed_hybrid_system.py

```diff
@@ -56,15 +57,25 @@ def nn_recognition(
             epochs = epochs if epochs is not None else list(checkpoints.keys())
 
             for pron, lm, prior, epoch in itertools.product(pronunciation_scales, lm_scales, prior_scales, epochs):
-                assert epoch in checkpoints.keys()
+                if epoch == "best":
+                    assert train_job is not None, "train_job needed to get best epoch checkpoint"
+                    best_checkpoint_job = GetBestPtCheckpointJob(
+                        train_job.out_model_dir, train_job.out_learning_rates, key="dev_loss_CE", index=0
```

Comment:
I know this might get messy, but can we always assume "dev_loss_CE" to exist like this here?

Reply:
Good point. Off the top of my head, I don't see a very clean and generic solution here. Any suggestions?

Reply:
It just has to be configurable, and I am fine having this as default

Reply:
```suggestion
                        train_job.out_model_dir, train_job.out_learning_rates, key=best_checkpoint_key, index=0
```